src/Tribe/Linked_Posts.php

```diff
@@ -1057,18 +1057,27 @@ public function saved_linked_post_dropdown( $post_type, $current = null ) {
 
 		add_filter( 'tribe_events_return_all_linked_posts_if_none', '__return_true' );
 
-		$my_linked_posts = $this->get_linked_post_info(
-			$post_type,
-			[
-				'post_status' => [
-					'publish',
-					'draft',
-					'private',
-					'pending',
-				],
-				'author'      => $current_user->ID,
-			]
-		);
+		$available_post_status = [
+				'publish',
+				'draft',
+				'private',
+				'pending',
+		];
+
+		/**
+		 *  Filters the available post statuses that are used to retrieve `my posts`.
+		 *
+		 * @since TBD
+		 *
+		 * @param array  $available_post_status Array of available post status. Example: publish, draft, private, pending
+		 * @param string $post_type Post type of the linked post
+		 */
+		$my_posts_post_status = apply_filters( 'tec_events_linked_posts_my_posts_post_status', $available_post_status, $post_type );
+
+		$my_linked_posts = $this->get_linked_post_info( $post_type, [
+						'post_status' => $my_posts_post_status,
+						'author'      => $current_user->ID,
+				] );
```

Comment:
Some indention tweaks:
```suggestion
		$my_linked_posts = $this->get_linked_post_info( $post_type, [
			'post_status' => $my_posts_post_status,
			'author'      => $current_user->ID,
		] );
```